php/helpers.php

```diff
@@ -18,14 +18,25 @@
  * @return mixed
  */
 function block_field( $name, $echo = true ) {
-	/*
-	 * Defined in Block_Lab\Blocks\Loader->render_block_template().
+	/**
+	 * An array containing the block attributes.
 	 *
-	 * @var array
+	 * @var array $attributes
 	 */
-	global $block_lab_attributes, $block_lab_config;
+	$attributes = block_lab()->loader->get_data( 'attributes' );
```

Comment:
This docblock was definitely helpful when there was a global, but do you think it's needed now that it's getting `$attributes` with a normal method call?

And likewise for a few other places in this PR.

Reply:
Done in https://github.com/getblocklab/block-lab/pull/435/commits/06d78c727f1a014a6905dd05bb64d2725795ca75